sandbox/user_guide.md

```diff
@@ -179,7 +178,7 @@ You can check the status of the job by entering the job number into the `jobId`
 
 The X-Progress header indicates the job's workflow status (Pending, In Progress, Completed, Archived, Expired, Failed). When in the In Progress state, an estimated completion percentage is appended to the X-Progress value (e.g., "In Progress (10%)").
 
-Once the job is completed, you will receive a `HTTP 200 Complete` response, which includes a URL ending in .ndjson, and an `encryptedKey` string. You’ll need the end of the URL in order to retrieve your data, and the `encryptedKey` in order to decrypt the file.
+Once the job is completed, you can download the file by clicking on the `Download` button, as shown in the image below.  You will have one hour before your token expires, and you will need to get another from token if it expires before you are finished interacting with the API.
```

Comment:
This might be a merge issue.  This should probably read something like:
```
Once the job is completed, you will receive a `HTTP 200 Complete` response, which includes a URL ending in .ndjson.  You’ll need the end of the URL in order to retrieve your data.
```

@epfrankle does this verbiage look ok?

Reply:
Same as above--thank you for catching this.